docs/source/developers/cpp/emscripten.rst

```diff
@@ -0,0 +1,100 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+.. highlight:: console
+
+.. _developers-cpp-emscripten:
+#################################################
+Cross compiling for WebAssembly with Emscripten
+#################################################
```

Comment:
It seems that we use different marker character (e.g. `#` for top level) for other documents such as https://raw.githubusercontent.com/apache/arrow/main/docs/source/developers/cpp/building.rst (e.g. `=` for top level) in the same directory.
Could you use the same style?